draft-02.txt

```diff
@@ -1315,6 +1321,12 @@ Internet-Draft     Application-Level Profile Semantics        March 2015
 
    Interoperability considerations:  None
 
+   Fragment identifier considerations:  Fragment identifiers used with
+      application/alps+json resources are simple, opaque strings that
```

Comment:
@mamund  How about:

opaque strings that do not contain any URL unsafe characters per http://www.rfc-base.org/txt/rfc-1738.txt which ...

I think we should then add some similar wording to the id definition. Thoughts?